Move prow-tools Dockerfile to prow/images (#2264)
kyma-project/test-infra: prow/images/prow-tools/README.md

# Prow Tools

## Overview

> The directory contains Dockerfile for prow tools image with prebuilt tools used in prow pipelines.

Reviewer: ```suggestion
The directory contains the Dockerfile for the prow tools image with prebuilt tools used in the prow pipelines.
```

> The image is used as a standalone image in several cleaners job and as additional dependency in `kyma-integration` images.

Reviewer: ```suggestion
The image is used as a standalone image in several cleaners jobs and as an additional dependency in the `kyma-integration` images.
```

I'm guessing it's `jobs` since there are `several`...? Correct me if I'm reading it wrong.

The image consists of:

> - prebuilt binaries from `development/tools/cmd` directory

Reviewer: ```suggestion
- prebuilt binaries from the `development/tools/cmd` directory
```

> ## Installation
> To build the Docker image, run this command:

Reviewer: ```suggestion
## Installation

To build the Docker image, run this command:
```
